pages/mednum/[slug].vue

```diff
@@ -0,0 +1,89 @@
+<script setup>
+import { ref } from 'vue'
+
+definePageMeta({ layout: 'edu' })
```

Comment:
Peut-etre nous devrions fait un refacto pour le nommage pour qu'il soit plus generic?
```suggestion
definePageMeta({ layout: 'tutoriel' })
```

Reply:
Je pense qu'on voudra faire un autre layout spécifique à la mednum mais je ne sais pas encore ce qu'on y mettra (écrit dans la section "Remarques" de la PR 😉).

Reply:
ahhh oui :)